specification/sbi_atee.adoc

```diff
@@ -13,6 +43,858 @@ running a VMM).
+| SBI_ERR_INVALID_PARAM | The `tvm_guest_id` was invalid.
+|===
+
+=== Function: TEE TVM Add Zero Pages (FID #3)
+[source, C]
+-----
+struct sbiret sbi_tee_tvm_add_zero_pages(unsigned long tvm_guest_id,
+                                         unsigned long base_page_address,
+                                         unsigned long tsm_page_type,
+                                         unsigned long num_pages,
+                                         unsigned long tvm_gpa);
+-----
+Maps num_pages zero-filled pages of confidential memory starting at page_addr into the TVM's
+address space starting at `tvm_gpa`. The tvm_guest_id is the unique identifier returned by
+a previous call to `sbi_tee_create_tvm()`. The `guest_physical_address` must lie within a region
+of confidential memory created with `sbi_tee_tvm_add_confidential_memory_region()`. 
+
+Zero pages may be added at any time, and are typically demand faulted on TVM access.
+
+The possible error codes returned in `sbiret.error` are shown below.
+
+[#table_sbi_tee_tvm_add_zero_pages_errors]
+.TEE TVM Add Zero Pages Errors
+[cols="2,3", width=90%, align="center", options="header"]
+|===
+| Error code            | Description
+| SBI_SUCCESS           | The zero-page mapping at `guest_physical_address` succeeded.
+| SBI_ERR_INVALID_PARAM | One or more parameters to the call were invalid.
+|===
+
+=== Function: TEE TVM Finalize (FID #4)
+[source, C]
+-----
+struct sbiret sbi_tee_tvm_finalize(unsigned long tvm_guest_id);
+-----
+Transitions the TVM specified by `tvm_guest_id` from the initializing state to a "runnable" state.
+The `tvm_guest_id` is the unique identifier returned by a previous call to `sbi_tee_create_tvm()`.
+
+Some TVM APIs like `sbi_tee_tvm_add_measured_pages()`, may not be called following a successful completion of
+this call.
+
+The possible error codes returned in `sbiret.error` are shown below.
+
+[#table_sbi_tee_tvm_finalize_errors]
+.TEE TVM Finalize Errors
+[cols="2,3", width=90%, align="center", options="header"]
+|===
+| Error code            | Description
+| SBI_SUCCESS           | The TVM specified by `tvm_guest_id` is now "runnable".
+| SBI_ERR_INVALID_PARAM | The `tvm_guest_id` was invalid.
+|===
+
+=== Function: TEE TVM VCPU Run (FID #5)
+[source, C]
+-----
+struct sbiret sbi_tee_tvm_vcpu_run(unsigned long tvm_guest_id,
+                                   unsigned long tvm_vcpu_id);
+-----
+Runs the VCPU specified by `tvm_vcpu_id` in the TVM specified by `tvm_guest_id`.
+The `tvm_guest_id` is the unique identifier returned by a previous call to`sbi_tee_create_tvm()`,
+and the `tvm_vcpu_id` must have been previously added using `sbi_tee_tvm_vcpu_create`.
+
+The `tvm_guest_id` must be in a "runnable" state. Note that the function does not
+return unless the TVM exits with a trap that cannot be handled by the TVM.
+
+[source, C]
+-----
+enum tvm_vcpu_exit_code {
+    /* The vCPU exited due to an interrupt directed at the host. */
+    host_interrupt = 0,
```

Comment:
Should enumerators be in `ALL_CAPS` to be consistent with other enums in the SBI spec?